docs/learn-accounts.md

```diff
@@ -0,0 +1,89 @@
+
+Alternatively, you might find some wallets on the [Polkaproject.com page](http://www.polkaproject.com/) but bear in mind that these are **unaudited** and not officially affiliated with Web3 Foundation or the Polkadot project unless otherwise stated.
+
+Hardware wallet integration from [Ledger](https://ledger.com) is coming soon.
+
+This is an [invalid external link](https://foobarbaznonexistent.com.uk) for testing our CI.
+
+This is an [invalid internal link](learn-something-that-doesnt-exist) for testing our CI.
+
+This is an [external 404 link](https://bitfalls.com/something-non-existent) for testing our CI.
+
+## Existential Deposit and Reaping
+
+When you generate an account (address), you only generate a _key_ which lets you access it. The account does not exist yet on-chain. For that, it needs the existential deposit - 0.01 KSM / DOT. This deposit is paid by the account sending you tokens. The one "creating" you pays for your creation.
+
+Going below 0.01 causes an account to be _reaped_. Your account will be wiped from the blockchain's state to conserve space. You do not lose access to the reaped address - as long as you have your private key or recovery phrase, you can still use the address - but it needs a top-up of another existential deposit to be able to interact with the chain.
+
+Here's another way to think about existential deposits. Ever notice those `Thumbs.db` files on Windows or `.DS_Store` files on Mac? Those are junk, they serve no specific purpose other than making previews a bit faster. If a folder is completely empty save for such a file, you can remove the folder to clear junk off your hard drive. That does not mean you lose access to this folder forever - you can always recreate it. You have the _key_, after all - you're the computer's owner. It just means you want to keep your computer clean until you maybe end up needing this folder again, and then recreate it. Your address is like this folder - it gets removed from the chain when nothing is in it, but gets put back when it has at least the existential deposit.
+
+## Indices
+
+A Kusama address can have an index. An index is like a short and easy to remember version of an address.
+
+Indices are populated in order. Think of them like slots going from 0 to any arbitrary number:
+
+`[0][1][2][3][4][5][6]...`
+
+If slots 0-2 are populated by addresses A, B and C respectively, and I add an existential deposit to address X, that address will automatically be put into slot 3. Henceforth, you can send me money by just sending to `[3]` rather than remembering my whole address.
+
+```js
+[0][1][2][3][4][5][6]...
+[A][B][C][X][ ][ ][ ]...
+```
+
+But what if an account gets reaped as explained above? In that case, the index is emptied. In other words, the slot frees up again. If someone creates a new account, they may end up using the same index another address was using before.
+
+Because of this unreliability of indices, they will **not be present** in Polkadot mainnet.
```

Comment:
Has this been confirmed? Last I heard it was "may" not be present.

Reply:
Last I heard was "probably won't be there". I say risk it and say "won't be" and fix later if needed.

Reply:
I would say keep it to "may" since if it does end up being included we don't want to be wrong. And we won't have to remember to come back to fix it.

Reply:
Ok